website/templates/project/addon/widget.mako

```diff
@@ -1,31 +1,33 @@
-<div class="panel panel-default" name="${short_name}">
+% if complete or 'write' in user['permissions']:
```

Comment:
There's a discussion on this line from https://github.com/CenterForOpenScience/osf.io/pull/5707/files - please read through it and comment on the functionality based on Nici's recommendations.